### ISSUE TITLE
Feature Request: Add option to define circle size based on a square size that fits within it

### ISSUE THREAD

**Issue by pittgoose** [donatj/Circle-Generator#6]
I'd love to work on this fix, but my idea is to say "I'd like to know what size circle I will need to fit around my 5x5 square".

This idea comes from me wanting to build an iron farm (25x25 square) inside of a flying saucer, and I don't really have a good tool to figure out what size circle I'll need. I'm sure I could look up the mathematical equations and do that math, but I'd like to look that up once, adapt it into your already wonderful circle tool, and then never have to do the math again.

Basically, I'd like to know if this feature ever has a hope of seeing the light of day in your project, or if I should fork your repo and adapt it into my own tool, or just start from scratch.

Thanks!

**Comment by pittgoose** [comment]
Looks like it's actually an easy equation. `Diameter of a circle = side of internal square * square root of 2`. I guess the work needed here is to actually fill those squares on the UI.
